lms/djangoapps/instructor/utils.py

```diff
@@ -159,7 +159,7 @@ def generate_course_forums_query(course_id, query_type, parent_id_check=None):
         {'$sort': SON([('_id.year', 1), ('_id.month', 1), ('_id.day', 1)])},
     ]
     if query_type == 'Comment':
-        if parent_id_check:
+        if parent_id_check is not None:
```

Comment:
this is the bugfix. parent_id_check is either true or false and I want it to get set as long as parent_id_check is not null